xmpp.go

```diff
@@ -1510,9 +1510,9 @@ type Chat struct {
 	Text    string
 	Subject string
 	Thread  string
-	Ooburl  string
-	Oobdesc string
-	Lang    string
+	// XEP-0066 Out-Of-Band url/desc
+	Oob  Oob
```

Comment:
On the one hand I like to combine OOB URL and description in it's own type, on the other hand this will be a breaking change.